### PR TITLE
Update applications index banner with RBD date information

### PR DESCRIPTION
**Pull request by steventux**
## Context

We'd (temporarily) like to convey information about how we calculate RBD dates over the xmas period.

## Changes proposed in this pull request

Adds information to the applications index banner.

image

Also added to the start page 

Screenshot 2020-11-27 at 14 49 46

## Link to Trello card
https://trello.com/c/D1Wggcij/3044-content-rbd-holiday-changes

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/views/provider_interface/application_choices/index.html.erb [hunk 1]
```diff
@@ -6,12 +6,11 @@
       <div class="app-banner" aria-labelledby="getting-ready-for-next-cycle-message">
         <div class="app-banner__message">
           <h2 class="govuk-heading-m" id="getting-ready-for-next-cycle-message">
```
Comment (benswannack) [inline app/views/provider_interface/application_choices/index.html.erb:8]: Not sure if this does anything, but we should probably change the id here

**Review by benswannack — APPROVED** [review]
Looks good

**Review by benswannack — CHANGES_REQUESTED** [review]
We are now also updating the banner on the start page to have the same content (which I will do now)

**Review by benswannack — APPROVED** [review]

**Review by duncanjbrown — COMMENTED** [review]
I think we need the margin

Screenshot 2020-11-27 at 14 33 24

**Review by duncanjbrown — APPROVED** [review]

**Pull request merged**